README.md

```diff
@@ -298,6 +298,12 @@ To do the same for the 10h16 skin, run the following Docker command correspondin
 
 If you want to debug problems in the Redux data flow, add the parameter `-e REDUX_LOG=on` to the command line above
 
+### Automatic recompilation of assets during development (wmde19)
+
+If you are working on the JavaScript files of the wmde19 skin and need automatic recompilation when a files changes, add the following setting to your `config.dev.json` file:
+
+	"assets-path": "http://localhostz:7072" 
```

Comment:
>localhostz
Probably a typo.

Reply:
Yeah, it's a typo, I'm no `1337 haxx0r` 😝